### PR TITLE
Recalculate fix

### PR DESCRIPTION
**Pull request by lucasgaspar**
Fixed an issue where the patterns were recalculated each time instead of only when the list changed

https://github.com/amco/unity-gesture-recognizer/issues/3

### CODE REVIEW

**Review by anaisrmzl — APPROVED** [review]
:shipit:

**Review by SrTartarus — APPROVED** [review]
:shipit:

**Review by Horothenic — APPROVED** [review]
:shipit:

File: README.md [hunk 1]
```diff
@@ -28,3 +28,8 @@ Just import the UnityPackage of the lastest version found on **builds** folder
      - Get the positions using the **GetPositions** function on the TrailRenderer
 - You can save backups of the gesture dataset using presets
 - See DemoScene for usage example
+# Change log
+- 1.0.1
+	- More performance, now it only recalculates the patterns when a change ocurred
```
Comment (giovas17) [inline README.md]: Try to be only more assertive in the Changelog, like ```Recalculation of patterns when a change occurred```, also occurred is misspelled.

**Review by giovas17 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lucasgaspar) [reply README.md]: Done

**Review by giovas17 — APPROVED** [review]

**Pull request merged**